features/FEATURE_BLE/targets/TARGET_CORDIO/stack_adaptation/pal_mbed_os_adaptation.cpp

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2018 Arm Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"){}
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mbed.h"
+#include "pal_types.h"
+#include "pal_uart.h"
+#include "pal_nvm.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* UART */
```

Comment:
Please remove these implementations if not needed :)

Reply:
I can't, we need to provide something for the linker, I will signpost and comment them